dashboard/db/schema.rb

```diff
@@ -1838,17 +1838,6 @@
     t.index ["key", "locale"], name: "index_videos_on_key_and_locale", unique: true
   end
 
-  add_foreign_key "ap_school_codes", "schools"
-  add_foreign_key "census_inaccuracy_investigations", "census_overrides"
-  add_foreign_key "census_inaccuracy_investigations", "census_submissions"
-  add_foreign_key "census_inaccuracy_investigations", "users"
-  add_foreign_key "census_overrides", "schools"
-  add_foreign_key "census_submission_form_maps", "census_submissions"
-  add_foreign_key "census_summaries", "schools"
-  add_foreign_key "circuit_playground_discount_applications", "schools"
-  add_foreign_key "hint_view_requests", "users"
-  add_foreign_key "ib_school_codes", "schools"
-  add_foreign_key "level_concept_difficulties", "levels"
```

Comment:
@bethanyaconnor I'm not sure why these disappear from our local DBs, but I think these are supposed to be in the schema. would you be willing to add these back?

Reply:
Yes, sorry. I don't know why my local environment does this -- I normally check but there was a merge conflict and I forgot 😞

Reply:
My local environment does this to me all the time too.